contribs/gmf/src/directives/partials/search.html

```diff
@@ -8,4 +8,5 @@
 <span class="clear-button ng-hide"
         ng-hide="!ctrl.clearButton || ctrl.input_value == ''"
         ng-click="ctrl.onClearButton()"></span>
+  <input data-ng-model="color" data-ng-change="ctrl.setStyleColor(color)"></input>
```

Comment:
This was part of a patch that @fgravin did for me.  I hadn't noticed it, but it shouldn't have been added.  You can remove this.